examples/basic_usage.ipynb

```diff
@@ -629,6 +629,32 @@
     "_Stay tuned for some new features!_"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e5e0c4ae",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "# Cross-validation with skore"
```

Comment:
- Please use "Heading 2" `##` style.
- Move the whole one block up so that "Stay tuned for some new features" remains the last element.

Reply:
done